### PR TITLE
[usage] reduce log noise

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Makes [this log message](https://console.cloud.google.com/logs/query;cursorTimestamp=2023-06-26T07:05:24.002063499Z;pinnedLogId=2023-06-26T07:05:24.002063499Z%2Fzprf54b5vovdbce3;query=resource.labels.container_name%3D%22usage%22%0A%22reset_usage%22%0Atimestamp%3D%222023-06-26T07:05:24.002063499Z%22%0AinsertId%3D%22zprf54b5vovdbce3%22?project=gitpod-191109) become info severity

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/usage/pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (c *Scheduler) Start() {
 				return jobErr
 			})
 			if err != nil {
-				if errors.Is(err, redsync.ErrFailed) {
+				if errors.Is(err, redsync.ErrTaken{}) {
```
Comment (easyCZ) [inline components/usage/pkg/scheduler/scheduler.go:73]: Let's downgrade this case to Info as well, and then have a general handler with Error for the case where not ErrTaken

**Review by easyCZ — APPROVED** [review]

File: components/usage/pkg/scheduler/scheduler.go (hunk 1 above)
Comment (svenefftinge) [reply components/usage/pkg/scheduler/scheduler.go:73]: I don't understand that comment. Can you explain?

Comment (easyCZ) [reply components/usage/pkg/scheduler/scheduler.go:73]: Ah, sorry. It's already info. I got confused with the `WithError` part. Ignore.

**Pull request merged**